src/main/java/mate/academy/spring/exception/AuthenticationException.java

```diff
@@ -1,6 +1,6 @@
 package mate.academy.spring.exception;
 
-public class AuthenticationException extends Exception {
+public class AuthenticationException extends RuntimeException {
```

Comment:
no need to change the extension
```suggestion
public class AuthenticationException extends Exception {
```